src/xcomm.cpp

```diff
@@ -187,6 +188,44 @@ namespace xpyt
         py::module kernel_module = py::module("kernel");
 
         py::class_<detail::xmock_object> _Mock(kernel_module, "_Mock");
+        py::class_<XInteractiveShell> XInteractiveShell(
```

Comment:
Naming convention:

- we use snake case in C++ (even for classes), so this line should probably changed into `py::class_<xinteractive_shell> interactive_shell(// same as before)

- names starting with `_` or `__` are reserved for use by the standard library.

Reply:
:+1:  but what do you do when you create a Python class wrapper for a C++ class? I guess you would use CamelCase for Python (like in the line above)?